CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+CHANGELOG
+Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
```

Comment:
We probably need a "2.12.0 Initial Release" line in this file somewhere. Not sure exactly where.

Reply:
Added on L6.